src/main/java/com/icthh/xm/ms/entity/config/DatabaseConfiguration.java

```diff
@@ -155,7 +158,7 @@ public LocalContainerEntityManagerFactoryBean entityManagerFactory(DataSource da
                                                                        CurrentTenantIdentifierResolver currentTenantIdentifierResolverImpl,
                                                                        LocalValidatorFactoryBean localValidatorFactoryBean) {
         Map<String, Object> properties = new HashMap<>();
-        properties.putAll(jpaProperties.getHibernateProperties(dataSource));
+        properties.putAll(jpaProperties.getHibernateProperties(new HibernateSettings()));
```

Comment:
please add properties.putAll(jpaProperties.getProperties());

Reply:
done

Reply:
in current implemetanion if we add custom propertie to spring.jpa.properties they all will be added.
So do not need to change something